### PR TITLE
add extra docs

### PR DESCRIPTION
**Pull request by Omarabdul3ziz**
- https://github.com/threefoldtech/tfgridclient_proxy/issues/234

### CODE REVIEW

**Comment by Omarabdul3ziz** [comment]
This PR should be updated after removing the RMB part

File: README.md [hunk 1]
```diff
@@ -1,186 +1,144 @@
...
-- install a chart
+- Create the service file
 
   ```bash
-  helm install gridproxy/gridproxy
+  cat << EOF > /etc/systemd/system/gridproxy-server.service
+  [Unit]
+  Description=grid proxy server
+  After=network.target
+  After=msgbus.service
```
Comment (mariobassem) [inline README.md]: is msgbus still needed??

File: docs/production.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  ```bash
+  chmod +x ./gridproxy-server
+  mv ./gridproxy-server /usr/local/bin/gridproxy-server
+  ```
+
+- Add a new systemd service
+
+```bash
+# create msgbus service
```
Comment (mariobassem) [inline docs/production.md]: is msgbus still needed?

**Review by mariobassem — COMMENTED** [review]
i only have one small comment, but in general lgtm...

**Review by mariobassem — APPROVED** [review]

**Pull request merged**